beta/azure-container-apps/ReadMe.md

```diff
@@ -60,95 +55,129 @@ Deploying 1Password SCIM Bridge on Azure Container Apps comes with a few benefit
-        ContainerAppName="op-scim-bridge-con-app"
+        ConAppEnv="op-scim-bridge-con-app-env"
+        ConAppName="op-scim-bridge-con-app"
         ```
 
     - Using PowerShell 
         ```pwsh
         $ResourceGroup="op-scim-bridge-rg"
         $Location="canadacentral"
-        $ContainerAppEnvironment="op-scim-bridge-con-app-env"
-        $ContainerAppName="op-scim-bridge-con-app"
+        $ConAppEnv="op-scim-bridge-con-app-env"
+        $ConAppName="op-scim-bridge-con-app"
         ```
 
-3. Create the resource group:
+5. Create the resource group: 
+
+    > **Note:** this step can be skipped if you are using an existing Resource Group to deploy your SCIM bridge to.
 
    ```
    az group create --name $ResourceGroup --location $Location
    ```
 
-4. Create the Container App Environment:
+6. Create the Container App Environment:
 
    ```
-   az containerapp env create --name $ContainerAppEnvironment --resource-group $ResourceGroup --location $Location
+   az containerapp env create --name $ConAppEnv --resource-group $ResourceGroup --location $Location --enable-workload-profiles false
    ```
 
-5. Upload your `scimsession` file to the Cloud Shell:
+7. Upload your `scimsession` file to the Cloud Shell:
    - Click the “Upload/Download files” button and choose Upload.
    - Find the `scimsession` file that you saved to your computer and choose it.
    - Make a note of the upload destination, then click Complete.
 
-6. Create the base Container App and Secret for your Base64-encoded `scimsession` credentials, _(using the bash or PowerShell syntax for the commands)_:
+8. Deploy the Container App Secret for the SCIM bridge. 
+
+    > **Note:** If you prefer to deploy the containers using the Azure Container App Commands, go to this section: [Azure ContainerApps Commands Deployment steps](#azure-container-apps-commands-deployment-steps), othersise follow the template file steps below. 
+    >
+
+    Create your Container App Secret:
     - Using bash
+        ```bash
+        az containerapp create --resource-group $ResourceGroup --environment $ConAppEnv --name $ConAppName  --secrets scimsession="$(cat /home/$USER/scimsession | base64)"
+        ```
+    - Using PowerShell 
+        ```pwsh
+        az containerapp create --resource-group $ResourceGroup --environment $ConAppEnv --name $ConAppName  --secrets scimsession="$([Convert]::ToBase64String([IO.File]::ReadAllBytes((Join-Path /home/$Env:USER/ 'scimsession'))) )"
+        ```
+
+9. Deploy your SCIM bridge containers based off the template file:
+
+    Upload the [aca-op-scim-bridge.yaml](aca-op-scim-bridge.yaml) file to the Cloud Shell:
+    - In a separate browser window, download the [aca-op-scim-bridge.yaml](aca-op-scim-bridge.yaml) (selecting the download icon from the top right) from our GitHub repository. 
+    - Click the “Upload/Download files” button and choose Upload.
+    - Find the `aca.op-scim-bridge.yaml` file that you saved to your computer and choose it.
```

Comment:
We could potentially have them curl the yaml down, save an upload/download step?